operations/mimir-mixin/config.libsonnet

```diff
@@ -46,10 +46,13 @@
 
     // The label used to differentiate between different Kubernetes clusters.
     per_cluster_label: 'cluster',
+    per_namespace_label: 'namespace',
+    per_job_label: 'job',
```

Comment:
Could you update the documentation at `docs/sources/operators-guide/monitor-grafana-mimir/requirements.md`? There's a table explaining which label names can be customised and which not. With this PR you're introducing a way to customize the namespace and job too.

Reply:
I've updated the documentation